src/node.js

```diff
@@ -116,10 +116,27 @@
 
       // If -i or --interactive were passed, or stdin is a TTY.
       if (process._forceRepl || NativeModule.require('tty').isatty(0)) {
+        var history;
+        var path = NativeModule.require('path');
+        var fs = NativeModule.require('fs');
+        var HISTORY_PATH = path.join(process.env.HOME || process.env.USERPROFILE,
```

Comment:
I honestly don't know this, but here is a pretty popular module https://github.com/sindresorhus/user-home and it does more stuff